ibis/expr/api.py

```diff
@@ -4032,6 +4032,72 @@ def _table_count(self):
     return ops.Count(self, None).to_expr().name('count')
 
 
+def _table_dropna(self, subset: Optional[List[str]] = None, how: str = 'any'):
+    """
+    Remove rows with null values from the table.
+
+    Parameters
+    ----------
+    subset : list of strings
+      Optional, columns names to consider. Defaults to all columns.
+    how : string
+      Determine whether a row is removed if there is at least one null
+      value in the row ('any'), or if all row values are null ('all').
+      Options are 'any' or 'all'. Default is 'any'.
+
+    Examples
+    --------
+    >>> import ibis
+    >>> t = ibis.table([('a', 'int64'), ('b', 'string')])
+    >>> t = t.dropna()  # Drop all rows where any values are null
+    >>> t = t.dropna(how='all')  # Only drop rows where all values are null
+    >>> t = t.dropna(subset=['a'], how='all')  # Only drop rows where all values in column 'a' are null  # noqa: E501
+
+    Returns
+    -------
+    table : TableExpr
+      New table expression
+    """
+    if subset is None:
+        subset = []
+    subset = util.promote_list(subset)
+    return ops.DropNa(self, how, subset).to_expr()
+
+
+def _table_fillna(self, replacements):
+    """
+    Fill null values in the table.
+
+    Parameters
+    ----------
+    replacements : scalar or dict
+      Value with which to fill the nulls. If passed as a dict, the
+      keys are column name strings that map to their replacement value.
+      If passed as a scalar, all columns are filled with that value.
+
+    Examples
+    --------
+    >>> import ibis
+    >>> t = ibis.table([('a', 'int64'), ('b', 'string')])
+    >>> t = t.fillna(0.0)  # Replace nulls in all columns with 0.0
+    >>> t.fillna({c: 0.0 for c, t in t.schema().items() if t == dt.float64})  # Replace all na values in all columns of a given type with the same value  # noqa: E501
+
+    Returns
+    -------
+    table : TableExpr
```

Comment:
add comment about type stability in Notes

Reply:
Added